docs/docs/03-language-reference.md

```diff
@@ -74,6 +74,15 @@ import TOCInline from '@theme/TOCInline';
 > let q: num? = nil;          // q is an optional num
 > ```
 
+Numeric literals can be formatted and padded with extra zeroes or underscores to make them easier to read.
+These don't affect the value of the number:
+
+> ```TS
+> let price = 0012.34;
```

Comment:
Not sure I've ever seen people wanting to intentionally add `0` padding before. Upon reading it, my first thought is to assume it's a mistake.
Not 100% against it, but curious about the value added here.

Reply:
I was thinking there might have been some use cases where you want to line up numbers in some way, though I don't feel strongly about it... I saw it in Swift so I figured it wouldn't do much harm.

```js
let prices = [
  00.50,
  02.57,
  14.99,
  07.06,
];

let ipAddresses = [
  [192, 168, 001, 001],
  [192, 168, 001, 010],
  [192, 168, 010, 001],
  [010, 000, 000, 001]
];
```